src/StaffWebUI/wwwroot/app.css

```diff
@@ -3,10 +3,6 @@ html.dark-mode {
     color-scheme: dark;
 }
 
-html:has(.ensure-page-scrollbar) {
-    overflow-y: scroll;
-}
-
```

Comment:
Why remove this?

Reply:
I've moved the scroll bar from being on the whole right side/page to just be on the tickets grid. So there is now basically one big sticky header with the title and tabs/filter.

Reply:
Can you still scroll with keyboard just using cursor keys when you first arrive at the page?

Reply:
No, you'd need to tab into the grid first. Works, but is more cumbersome than how it was originally. 

I can change it back to original behavior, but then I can't get the header on the `FluentDataGrid` to stick. That is annoying but far less intrusive then not being able to use cursor keys immediately. You probably do not want to start a sort when you're half way down the list of tickets.

Reply:
Do you mean the column headers would become invisible once you start to scroll down?

Reply:
Yes:
![image](https://github.com/user-attachments/assets/5fef7b66-2b4f-44de-a2d2-6a9d4de35f52)

Reply:
Sorry, I don't think we can do that. Losing track of what the columns mean harms the usability too much.